### PR TITLE
Consistently use SpdxConstants.NONE and SpdxConstants.NOASSERTION

### LINKED ISSUE #2781: Remove NONE and NOASSERTION constants from SpdxLicense
> since these constants are not exclusively used for license properties

If that's the case I'd actually drop them from `SpdxLicense` instead, and make existing code use the constants from `SpdxConstants` (which could in the same run be moved to a more central place). But I'm fine doing this as a follow-up PR.

_Originally posted by @sschuberth in https://github.com/oss-review-toolkit/ort/pull/2773_

### PR DESCRIPTION
**Pull request by fviernau**
Fixes #2781.

### CODE REVIEW

File: spdx-utils/src/main/kotlin/SpdxConstants.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * License-Filename: LICENSE
  */
 
-package org.ossreviewtoolkit.spdx.model
+package org.ossreviewtoolkit.spdx
```
Comment (sschuberth) [inline spdx-utils/src/main/kotlin/SpdxConstants.kt:20]: Commit message: s/conist/consist/

File: spdx-utils/src/main/kotlin/SpdxConstants.kt [hunk 2]
```diff
@@ -35,5 +35,8 @@ object SpdxConstants {
      */
     const val NOASSERTION = "NOASSERTION"
 
-    internal fun isNotPresent(value: String) = value in setOf(NONE, NOASSERTION)
+    internal fun isNotPresent(value: String) = value in setOf(
+        NONE,
+        NOASSERTION
+    )
```
Comment (sschuberth) [inline spdx-utils/src/main/kotlin/SpdxConstants.kt:41]: Unrelated code change (that also does not seem to be necessary / improve readability anyway).

File: analyzer/src/main/kotlin/managers/Npm.kt [hunk 3]
```diff
@@ -56,7 +56,7 @@ import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.createAndLogIssue
 import org.ossreviewtoolkit.model.jsonMapper
 import org.ossreviewtoolkit.model.readValue
-import org.ossreviewtoolkit.spdx.SpdxLicense
+import org.ossreviewtoolkit.spdx.SpdxConstants
```
Comment (sschuberth) [inline analyzer/src/main/kotlin/managers/Npm.kt:59]: I'd appreciate a generalization of the commit message like this:

```
Use NONE / NOASSERTION from SpdxConstants instead of SpdxLicense

These constants are not only used for licenses, so use them from
SpdxConstants as a preparation for eliminating them from SpdxLicense
later.
```

Comment (sschuberth) [reply analyzer/src/main/kotlin/managers/Npm.kt:59]: Also, I believe this misses to replace `NOASSERTION` in `scanner/src/main/kotlin/scanners/ScanCode.kt`.

File: spdx-utils/src/main/kotlin/SpdxConstants.kt (hunk 1 above)
Comment (fviernau) [reply spdx-utils/src/main/kotlin/SpdxConstants.kt:20]: Done.

File: spdx-utils/src/main/kotlin/SpdxConstants.kt (hunk 2 above)
Comment (fviernau) [reply spdx-utils/src/main/kotlin/SpdxConstants.kt:41]: Done.

File: analyzer/src/main/kotlin/managers/Npm.kt (hunk 3 above)
Comment (fviernau) [reply analyzer/src/main/kotlin/managers/Npm.kt:59]: Done.

**Comment by fviernau** [comment]
Merged regardless of appveyor as it actually was successful. The result status did not get reflected most likely due to the github outage.
